airbyte-integrations/connectors/source-woocommerce/source_woocommerce/woocommerce.yaml

```diff
@@ -0,0 +1,300 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: [ ]
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_woocommerce/schemas/{{ options['name'] }}.json"
+  requester:
+    url_base: "https://{{ config['shop'] }}/wp-json/wc/v3"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_key'] }}"
+      password: "{{ config['api_secret'] }}"
+    request_options_provider:
+      request_parameters:
```

Comment:
In the original python code, we were appending user agent and format headers. We have a field `request_headers` to allow for that. Are they no longer required to make requests to the API?